CONTRIBUTING.md

```diff
@@ -145,17 +145,22 @@ that you know.
 The below is an excerpt from the types for the `datetime` module.
 
 ```python
-MAXYEAR = ...  # type: int
-MINYEAR = ...  # type: int
+MAXYEAR: int = ...
```

Comment:
@JelleZijlstra suggested that ``...`` is not needed here (which makes sense since module constants always have a value)